mozartdata/transforms/bridge/orders.sql

```diff
@@ -362,6 +390,14 @@ FROM dim.orders orders
 							aggregate_netsuite.channel_currency_id_ns = cer.transaction_currency_id_ns
 		 LEFT OUTER JOIN fulfillment_info
 						 ON fulfillment_info.order_id_edw = orders.order_id_edw
+         left join
+            aftership_rmas             as aftership_return_orders -- orders that have return or refund rmas
+            on
+                orders.order_id_edw = aftership_return_orders.original_order_id_edw
+         left join
+            aftership_orders_from_rmas as aftership_exchange_orders -- orders that have exchange or warranty rmas
+            on
+                orders.order_id_edw = aftership_exchange_orders.rma_exchange_order_id_edw
```

Comment:
aliasing aliases? 

Let's just change the CTE names if those are your preferred table aliases
```suggestion
            aftership_return_orders -- orders that have return or refund rmas
            on
                orders.order_id_edw = aftership_return_orders.original_order_id_edw
         left join
            aftership_exchange_orders -- orders that have exchange or warranty rmas
            on
                orders.order_id_edw = aftership_exchange_orders.rma_exchange_order_id_edw
```

Reply:
done